neqo-transport/tests/sim/taildrop.rs

```diff
@@ -171,8 +183,22 @@ impl Node for TailDrop {
 
     fn print_summary(&self, test_name: &str) {
         println!(
-            "{}: taildrop: rx {} drop {} tx {} maxq {}",
-            test_name, self.received, self.dropped, self.delivered, self.maxq,
+            "{}: taildrop: rx {} drop {} tx {} maxq {} thruoghput {}",
```

Comment:
You should calculate throughput in a separate step.  Also:

```suggestion
            "{}: taildrop: rx {} drop {} tx {} maxq {} throughput {}",
```